manager-parent/manager-core/src/test/resources/application.properties

```diff
@@ -37,4 +37,8 @@ event-bridge.resources.workers.job-retry-interval-seconds=1
 # Disable error processing scheduler
 event-bridge.processing-errors.cleanup.schedule=0 0 0 1 1 ? 2099
 
+# LOGGING
+# Use unstructured logging in test
+quarkus.log.console.json=false
```

Comment:
I moved the `TEST` override to `src/main/test/resources/application.properties` for consistency with other test overrides.